### PR TITLE
Bug 1636292 - fix importing of multiple-buffer base64 encodings

### PR DESCRIPTION
**Pull request by djmitche**
Bugzilla Bug: [1636292](https://bugzilla.mozilla.org/show_bug.cgi?id=1636292)

This was really pretty silly of me.  I assumed that entities would concatenate the buffers and then base64-decode, when in fact it base64-decodes, then concatenates.

This is a draft because, unfortunately, it's "cheating" to fix this in an older DB version, and for example staging won't re-run the version-8 migration script so it won't see a benefit from this change.  How should we handle this?  @helfi92 what do you think?

### CODE REVIEW

**Comment by djmitche** [comment]
That makes sense!

**Comment by edunham** [comment]
I can manually re-run arbitrary migrations for you to undo problems caused by a bad release to staging.

**Comment by djmitche** [comment]
To do so would require downgrading from version 9 to 8 after rolling services back, too.  We want to avoid relying on extraordinary actions by ops folks if possible!

the generic-worker failures here are #2783

File: db/test/versions/huge_bufs.js [hunk 1]
```diff
@@ -0,0 +1,2207 @@
+// Data from a real DB.  See 0008_test.js
```
Comment (helfi92) [inline db/test/versions/huge_bufs.js:1]: nit: I would probably put this inside `versions/fixtures` to keep things clean in that directory.

**Pull request merged**